### ISSUE TITLE
`bindgen --version` does not work (0.71.0 regression)

### ISSUE THREAD

**Issue by xry111** [rust-lang/rust-bindgen#3039]
```
$ bindgen --version
error: the following required arguments were not provided:
  <HEADER>

Usage: bindgen <FLAGS> <OPTIONS> <HEADER> -- <CLANG_ARGS>...

For more information, try '--help'.
```

This breaks building Mesa of which the build system checks bindgen rust-version using the command.

It worked fine with 0.70.1, and I don't think it makes too much sense to mandate a header when checking rust-bindgen version.  Thus I consider it a bug.

**Comment by ojeda** [comment]
Yeah (https://github.com/rust-lang/rust-bindgen/issues/3037, https://github.com/rust-lang/rust-bindgen/issues/2677#issuecomment-1789663526).
